kalite/contentload/management/commands/unpack_assessment_zip.py

```diff
@@ -51,19 +51,27 @@ def handle(self, *args, **kwargs):
             return
 
         if is_valid_url(ziplocation):  # url; download the zip
-            logging.warn("Downloading assessment item data from a remote server. Please be patient; this file is big, so this may take some time...")
+            print "Downloading assessment item data from a remote server. Please be patient; this file is big, so this may take some time..."
             # this way we can download stuff larger than the device's RAM
             r = requests.get(ziplocation, prefetch=False)
+            content_length = r.headers.get("Content-Length")
+            print "Downloaded size: ", str(int(content_length) // 1024 // 1024) + " MB" if content_length else "Unknown"
+            sys.stdout.write("Downloading file...")
+            sys.stdout.flush()
             f = tempfile.TemporaryFile("r+")
-
-            for chunk in r.iter_content(chunk_size=1024): 
+            for cnt, chunk in enumerate(r.iter_content(chunk_size=1024)):
                 if chunk: # filter out keep-alive new chunks
                     f.write(chunk)
+                    if cnt % 1000 == 0:
+                        sys.stdout.write(".")
+                        sys.stdout.flush()
```

Comment:
I have no preference, but note that you can suppress the newline with `print` with a trailing comma:
`print ".",`